cli_reference/kn-cli-tools.adoc

```diff
@@ -0,0 +1,35 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="kn-cli-tools"]
+= Knative CLI (`kn`) for use with OpenShift Serverless
+:context: kn-cli-tools
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+`kn` extends the functionality of the `oc` or `kubectl` tools to enable interaction with Knative components on {product-title}.
+
+You can enable Knative on {product-title} by installing {ServerlessProductName}. For more information, see the documentation on xref:../serverless/serverless-getting-started.adoc#serverless-getting-started[Getting started with {ServerlessProductName}].
+
+[NOTE]
+====
+{ServerlessProductName} cannot be installed using the `kn` CLI. A cluster administrator must install the {ServerlessOperatorName} and set up the Knative components, as described in the _Serverless applications_ documentation for {product-title}.
+====
+
+== Key features
+`kn` is designed to make serverless computing tasks simple and concise, and has the following key features:
+
+* Full support for managing all features of Knative Serving, such as services, revisions, and traffic splitting
+* Easily create and manage Knative Eventing components, such as event sources, brokers, and triggers for filtering events.
++
+[NOTE]
+====
+Knative Eventing is currently available as a Technology Preview feature of {ServerlessProductName}.
+====
+* A plugin architecture similar to that of `kubectl` plugins
+* A thin, client-specific API in golang, that enables tasks such as synchronously waiting for Knative service write operations
```

Comment:
I don't think the API aspect is relevant for this scope (its about end-user, not developers). Maybe we can rephrase this that `kn` adds specific support for interactive and scripted usage like waiting synchronously on the results of an operations ?